docs/account/email-connection.md

```diff
@@ -23,6 +23,22 @@ You can also define more than one email address for a module and send notificati
 
 <Screenshot url='https://cdn.appcircle.io/docs/assets/email-notify-signing-identity.png' />
 
+:::info
+After completing the specified action in Appcircle, you have the option to share release notes via email.
+To enable this feature, ensure you include the [**Publish Release Notes**](https://docs.appcircle.io/integrations/managing-release-notes/) step in your workflow.
+
+Additionally, note that you can access download links for the release notes for a duration of 90 days.
+:::
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/2446-ReleaseNotesViaEmail.png' />
+
+:::info
+After completing the specified action in Appcircle, you have the option to share the test results via email.
+To enable this feature, ensure you include the [**Test Reports**](https://docs.appcircle.io/continuous-testing/running-ios-unit-and-ui-tests#generating-test-report) step in your workflow.
+:::
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/2446-TestReportsViaEmail.png' />
```

Comment:
@csonuryilmaz @atkncamurcu The warning message "Cannot Connect to server" appears in the screen shot background. The screenshot may need to be changed. If it is not necessary, I fixed some typos and added related documentation links to mentioned workflow components.

Reply:
@boztopuz We will improve the SS after the PR merge by overwriting the SS with the fixed image.

Thanks for mentioning the case. :pray: